contexts/credentials/v2

```diff
@@ -55,6 +55,15 @@
         }
       }
     },
+    "_sd_alg": {
+      "@id": "https://www.iana.org/assignments/jwt#_sd_alg"
+    },
+    "_sd": {
+      "@id": "https://www.iana.org/assignments/jwt#_sd"
+    },
+    "...": {
```

Comment:
"..." is a registered claim name?

Reply:
answering my own questions... yep: https://www.ietf.org/archive/id/draft-ietf-oauth-selective-disclosure-jwt-05.html#section-5.1

Reply:
haha yes : ) its how they annotate "array" values.